code/modules/wiremod/components/hud/click_intercept.dm

```diff
@@ -0,0 +1,64 @@
+/**
+ * # Click Intercept Component
+ *
+ * When activated intercepts next click and outputs clicked atom.
+ * Requires a BCI shell.
+ */
+
+/obj/item/circuit_component/click_intercept
```

Comment:
Rename the path and comments and everything to target_intercept as well to remove confusion.